README.adoc

```diff
@@ -0,0 +1,88 @@
+= API Library
+
+image:https://img.shields.io/badge/vert.x-4.3.2-purple.svg[link="https://vertx.io"]
+image:https://github.com/cyface-de/api/workflows/Cyface%20API/badge.svg[link="https://github.com/cyface-de/api/actions"]
+
+This library is used by the Cyface Vert.X APIs like the https:///github.com/cyface-de/data-collector[Cyface Data Collector] software.
+
+It contains share code required by most of the APIs, like authentication and setup.
+
+If you require this software under a closed source license for you own projects, please https://www.cyface.de/#kontakt[contact us].
+
+Changes between versions are found in the link:https://github.com/cyface-de/api/releases[Release Section].
+
+The project uses link:https://gradle.org/[Gradle] as the build system.
+It is set up as a Gradle multi project build.
+There is one general categories of sub-projects.
+The libraries shared by several Cyface executables which are grouped in `libs`.
```

Comment:
Since there are no executables here, why bother with an additional nesting? Only makes the project setup more complex.

Reply:
TO-DO